.github/workflows/checksums-of-release-draft.yml

```diff
@@ -0,0 +1,34 @@
+name: Checksums of release draft
+
+on:
+  workflow_run:
+    workflows:
+      - Package Neuron for Release Draft
+    types:
+      - completed
+
+jobs:
+  checksums:
+    runs-on: ubuntu-latest
+    if: github.event.workflow_run.conclusion == 'success'
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v3
+
+      - name: Setup Node
+        uses: actions/setup-node@v3
+        with:
+          node-version: 'latest'
+
+      - name: Download binaries
+        run: |
+          mkdir -p /tmp/Neuron
+          node ./scripts/download-binaries-from-release-draft.js /tmp/Neuron
+        shell: pwsh
+        env:
+          TOKEN: ${{ secrets.GITHUB_TOKEN }}
+          REPO: ${{ github.repository }}
+
+      - name: Checksums
+        run: node ./scripts/generate-checksum-table.js /tmp/Neuron
```

Comment:
I found that the script finishes after writing a `checksum.txt` without an upload to GitHub action. Is it expected?

Reply:
Yes, the `checksum.txt` is an add-on for custom distribution, it will be generated in the directory where binaries are, so the directory can be published directly(e.g., google driver) and users may check the checksum table online.

In this workflow, the checksum table will be sent to the stdout so I can copy&paste it into the release note.

When we add an action to generate a release note automatically, the checksum table in stdout can be redirected to the next workflow.

Reply:
thx for your explanation, the distribution workflow will still evolve later, and this is not the final version